dashboard/app/models/user.rb

```diff
@@ -246,6 +247,19 @@ def delete_course_as_facilitator(course)
     courses_as_facilitator.find_by(course: course).try(:destroy)
   end
 
+  def log_admin_save
+    # Do not log for development or adhoc environments.
+    return unless [:test, :staging, :levelbuilder, :production].include? rack_env
+
+    ChatClient.message 'infra-security',
+      "#{admin ? 'Granting' : 'Revoking'} UserPermission: "\
+      "environment: #{rack_env}, "\
+      "user ID: #{id}, "\
+      "email: #{email}, "\
+      "permission: ADMIN",
+      color: 'red'
```

Comment:
I also think of red as urgent. How about a compromise, like orange, so it stands out but doesn't appear urgent.